libavfilter/dnn/dnn_backend_native_layer_conv2d.c

```diff
@@ -33,13 +34,25 @@ typedef struct thread_common_param{
     const void *parameters;
     NativeContext *ctx;
     int thread_num;
+    int step;
```

Comment:
do we have a bettering naming for 'step'?  Or step is already a good name for the cpu flags?